tests/main/interfaces-cifs-mount/task.yaml

```diff
@@ -0,0 +1,66 @@
+summary: Ensure that the cifs-mount interface works.
+
+details: |
+    The cifs-mount interface allows mounting and unmounting CIFS filesystems.
+
+prepare: |
+    snap install test-snapd-cifs-mount
```

Comment:
Shouldn't this be installing the local snap?

Reply:
As I need some dependencies as part of this snap, I had to create a snap and upload it to the store.